cribl-edge/config/log.sls

```diff
@@ -0,0 +1,36 @@
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+
+{%- from tplroot ~ "/map.jinja" import mapdata as cribl_edge with context %}
+
+Cribl-Edge Nuke Default Log-Dir:
+  file.absent:
+    - name: '{{ cribl_edge.package.install_path }}/log'
+    - onlyif:
+      - '[[ -d {{ cribl_edge.package.install_path }}/log ]]'
+    - require:
+      - file: Cribl-Edge OS Log-Dir Setup
+
+Cribl-Edge OS Log-Dir Setup:
+  file.directory:
+    - group: 'root'
+    - mode: '0700'
+    - name: '/var/log/cribl-edge'
+    - require:
+      - archive: 'Cribl-Edge Archive Extracted'
```

Comment:
Will run into a bit of an ordering issue, if this state is called directly, e.g. `salt-call state.sls cribl-edge.config.log`. The way to work around that problem is to include the sls that defines this "archive" state. For an example, see this setup in the forescout formula:

https://github.com/plus3it/forescout-secure-connector-formula/blob/35705d6f49464956a79d67455c797c39d03a999f/forescout-secure-connector/service/running.sls#L3-L8

Reply:
Ok, I _think_ we're good on this. I ran:

1. `salt-call -c /opt/watchmaker/salt/ state.apply cribl-edge`
1. `salt-call -c /opt/watchmaker/salt/ state.apply cribl-edge.clean`
1. `salt-call -c /opt/watchmaker/salt/ state.apply cribl-edge.config.log`

With no errors.